include/roamer.h

```diff
@@ -25,38 +25,38 @@ typedef struct ROAMER {
     u8 status;
     u8 active;
     u8 dummy;
-} ROAMER;
+} Roamer;
 
-typedef struct ROAMER_SAVE {
+typedef struct RoamerSaveData {
     u32 rand[2];
     u32 playerLocationHistory[2];
-    ROAMER data[ROAMER_MAX];
+    Roamer data[ROAMER_MAX];
     u8 unk_60[ROAMER_MAX];
     u8 unk_64;
     u8 repelSteps;
     u8 unk_66;
     u8 flutePlayed;
-} ROAMER_SAVE;
+} RoamerSaveData;
 
 u32 Save_Roamers_sizeof(void);
-void Save_Roamers_Init(ROAMER_SAVE *roamer);
-void Roamers_SetRand(ROAMER_SAVE *roamer, u32 param);
-u32 Roamers_GetRand(ROAMER_SAVE *roamer, u32 which);
-ROAMER_SAVE *Save_Roamers_Get(SAVEDATA *saveData);
+void Save_Roamers_Init(RoamerSaveData *roamer);
+void Roamers_SetRand(RoamerSaveData *roamer, u32 param);
+u32 Roamers_GetRand(RoamerSaveData *roamer, u32 which);
+RoamerSaveData *Save_Roamers_Get(SAVEDATA *saveData);
 void RoamerSave_SetOutbreakActive(SAVEDATA *saveData);
-u8 RoamerSave_OutbreakActive(ROAMER_SAVE *roamerSave);
-void PlayerLocationHistoryPush(ROAMER_SAVE *roamerSave, u32 mapsec);
-u32 PlayerLocationHistoryGetBack(ROAMER_SAVE *roamerSave);
-u8 Roamer_GetLocation(ROAMER_SAVE *roamerSave, int a1);
-void Roamer_SetLocation(ROAMER_SAVE *roamerSave, int a1, u8 a2);
-u8 GetRoamerIsActiveByIndex(ROAMER_SAVE *roamerSave, int a1);
-void RoamerMon_Init(ROAMER ** roamer_p);
-ROAMER *Roamers_GetRoamMonStats(ROAMER_SAVE *roamerSave, int a1);
-int GetRoamerData(ROAMER *roamer, int a1);
-void SetRoamerData(ROAMER *roamer, int a1, int val);
-u8 *RoamerSave_GetRepelAddr(ROAMER_SAVE *roamerSave);
-BOOL RoamerSave_RepelNotInUse(ROAMER_SAVE *roamerSave);
-void RoamerSave_SetFlute(ROAMER_SAVE *roamerSave, u8 a1);
-u8 RoamerSave_GetFlute(ROAMER_SAVE *roamerSave);
+u8 RoamerSave_OutbreakActive(RoamerSaveData *roamerSave);
+void PlayerLocationHistoryPush(RoamerSaveData *roamerSave, u32 mapsec);
+u32 PlayerLocationHistoryGetBack(RoamerSaveData *roamerSave);
+u8 Roamer_GetLocation(RoamerSaveData *roamerSave, int a1);
+void Roamer_SetLocation(RoamerSaveData *roamerSave, int a1, u8 a2);
+u8 GetRoamerIsActiveByIndex(RoamerSaveData *roamerSave, int a1);
+void RoamerMon_Init(Roamer ** roamer_p);
+Roamer *Roamers_GetRoamMonStats(RoamerSaveData *roamerSave, int a1);
```

Comment:
Romaers->RoamerSave(Data)?

Reply:
you mean for the func names?

Reply:
yeah, since having Roamer_ and Roamers_ prefixes don't make that much sense

Reply:
which ones specifically do you think should be changed?

Reply:
Roamers_ seems to refer to relating to the RoamerSaveData struct right? Where Roamer_ refers to the Roamer struct?

Reply:
ok, let me rephrase, For Save_Roamers_Init, what do you think would be best to replace that with